CHANGELOG.md

```diff
@@ -6,6 +6,84 @@ All notable changes to the [Camunda Modeler](https://github.com/camunda/camunda-
 
 ___Note:__ Yet to be released changes appear here._
 
+## 5.1.0
+
+### Main changes
+
+* Multi-element context pad added to our tools for quick deletion, alignment, and spacing
+* Telemetry migrated to Mixpanel
+* BPMN and DMN editors have received UX improvements:
+	* clearer selection and multiselection
+	* input fields in the properties panel no longer lose changes
+* Conditional element template properties available in C8 
+* Stability and QOL improvements across the application
```

Comment:
Not sure what QOL is. Also, is this about any specific changes? Let's make it explicit or remove it.